onnxruntime/core/providers/dml/DmlExecutionProvider/src/External/DirectMLHelpers/ApiTraits.h

```diff
@@ -2532,6 +2630,9 @@ inline gsl::czstring ToString(DML_OPERATOR_TYPE value)
     case DML_OPERATOR_ELEMENT_WISE_QUANTIZED_LINEAR_ADD: return "DML_OPERATOR_ELEMENT_WISE_QUANTIZED_LINEAR_ADD";
     case DML_OPERATOR_ROI_ALIGN_GRAD: return "DML_OPERATOR_ROI_ALIGN_GRAD";
     case DML_OPERATOR_BATCH_NORMALIZATION_TRAINING: return "DML_OPERATOR_BATCH_NORMALIZATION_TRAINING";
+    case DML_OPERATOR_RESAMPLE2: return "DML_OPERATOR_RESAMPLE2";
+    case DML_OPERATOR_RESAMPLE_GRAD1: return "DML_OPERATOR_RESAMPLE_GRAD1";
```

Comment:
Missing the new activation functions? (Has this file diverged from what we have in WAI?)

Reply:
@sumitsays: Do you recall adding Gelu here or not?

Reply:
@adtsai Actually it's evidently a bug in `dml\Tools\GenerateSchema.ps1`, as it's not present in the DML side either. For some reason in DirectMLSchema.json, all our operators are under "operators:", *except* the activation operators which are under "activationOperators:", except for `ACTIVATION_RELU_GRAD` which is actually under "operators" 🙃. I don't know why activationOperators are deemed any more special than elementwise or reduction to deserve their entire own category, but I feel they should all just be put under `operators`, and we add a flag to the entries like `isActivation: true`, like we have for other flags (e.g. `"dimensionsGenericAndMatching": true` or `"private": true`...).

So I'll leave the `switch` as-is, since it's not causing any issues, and it leaves the two headers consistent (but you're right about the union missing GELU).